templates/job-submit.php

```diff
@@ -56,7 +56,7 @@
 			<input type="hidden" name="job_manager_form" value="<?php echo $form; ?>" />
 			<input type="hidden" name="job_id" value="<?php echo esc_attr( $job_id ); ?>" />
 			<input type="hidden" name="step" value="<?php echo esc_attr( $step ); ?>" />
-			<input type="submit" name="submit_job" class="button" value="<?php esc_attr_e( $submit_button_text ); ?>" />
+			<input type="submit" name="submit_job" class="button" value="<?php echo esc_attr( $submit_button_text ); ?>" />
```

Comment:
$submit_button_text is already passed through translation calls here: https://github.com/Automattic/WP-Job-Manager/blob/master/includes/forms/class-wp-job-manager-form-edit-job.php#L90 so this should be fine.